src/components/Offramp/Confirm.view.tsx

```diff
@@ -68,6 +68,45 @@ export const OfframpConfirmView = ({
     //////////////////////
     // state and context vars for claim link offramp
 
+    //////////////////////
+    // utility JSX vars
+
+    // type: 'iban' or other
+    // TODO: standardize this type
+    let accountType = user?.accounts.find((account) => account.account_identifier === offrampForm.recipient)?.account_type
+
+    // TODO: remove hardcoding fee amounts from here and refactor to consts file
```

Comment:
praise: good todo comment.

These should prob be pulled from `Cashout.consts.ts`